### ISSUE TITLE
Screen Reader: Many Pages: The error message is not associated with the field

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76939]
## Action Performed:
1. Using JAWS + Chrome, open the page https://new.expensify.com/
2. Navigate using TAB to the phone or email input field.
3. Trigger an error message, then press Tab/Shift+Tab key to focus the field with an error.
4. Note how the field is announced.

Other Occurrences:
1. On Settings - Save the world - I know a teacher
2. On Settings - Save the world - Intro your school principal
3. On Settings - Security - Close account
4. On Settings - Security - Two-factor authentication (step 2 of 2)
5. On Settings - Profile - Display name
6. On Settings - Profile - DOB
7. On Settings - Profile - Phone number
8. On Settings - Profile - Legal name
9. On Settings - Profile - Address
10. On Workspaces - Duplicate Workspaces
11. On Workspaces - Overview - Workspace Name
12. On Workspace - Categories - Add Category
13. On Workspace - Expensify Card - Bank info
14. On Workspace - Create Workspace - Confirm Workspace
15. On Create Report - Add payment card

## Expected Result:
The error message should be associated with the form control, so that it is announced upon focusing the control.

## Actual Result:
The error message is not associated with the form control and is not announced upon focusing the control.

HTML:
<div class=""css-g5y9jx r-18u37iz r-1awozwy r-knv0ih r-15zivkp""><div class=""css-g5y9jx"" data-testid=""""><svg xmlns=""[http://www.w3.org/2000/svg""](http://www.w3.org/2000/svg%22%22) xml:space=""preserve"" viewBox=""0 0 20 20"" width=""20"" height=""20"" hovered=""false"" pressed=""false"" fill=""#F25730""><circle cx=""10"" cy=""10"" r=""7.5""></circle></svg></div><div class=""css-g5y9jx r-13awgt0 r-1jkjb""><div dir=""auto"" class=""css-146c3p1"" style=""color: rgb(242, 87, 48); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; line-height: 16px; margin-bottom: 0px;"">Please enter an email or phone number</div></div></div>"

## Area issue was found in:
Landing Page

## Failed WCAG checkpoints
WCAG 1.3.1 Info and Relationships
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

## User impact:
Users with different disabilities employ assistive technology to adapt content according to their specific needs. For example, users who are blind (using a screen reader) benefit when information conveyed through color is also available in text (including text alternatives for images that use color to convey information), and users who are deaf-blind and use braille (text) refreshable displays may be unable to access color-dependent information.

## Suggested resolution:
Use the aria-describedby to associate the error message with the form control.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/9368e676-9f73-468c-96b6-a2e30cd31143

<img width="1917" height="1001" alt="Image" src="https://github.com/user-attachments/assets/036bc222-8cba-4dad-bf82-ff2f84065203" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997302
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @rushatgabhane</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @rushatgabhane</details>

**Comment by suneox** [comment]
This issue still depends on https://github.com/Expensify/App/issues/77572, and we’re still working on that [PR](https://github.com/Expensify/App/pull/81679).

**Comment by mallenexpensify** [comment]
@marufsharifi paid, summary updated above. Thx.

@rushatgabhane 👀  plz, I'm unsure what's best. 
> @rushatgabhane Do you agree that we should handle the payment for this issue separately, as it uses a different solution as https://github.com/Expensify/App/issues/79229#issuecomment-3806330629?

**Comment by trjExpensify** [comment]
approved $250 for @suneox
